source/usage-examples/insertMany.txt

```diff
@@ -4,14 +4,14 @@ Insert Multiple Documents
 
 .. default-domain:: mongodb
 
-You can insert multiple documents using the :node-api:`insertMany()
+You can insert multiple documents using the :node-api:`collection.insertMany()
 <Collection.html#insertMany>` method. The ``insertMany()`` takes an array
 of documents to insert into the specified collection.
 
-You can create an :mdn:`Object
-<Web/JavaScript/Reference/Global_Objects/Object>` to specify additional
-options. Specify ``ordered:true`` to prevent inserting the remaining
-documents if the insertion failed for a previous document in the array.
+You can specify additional options in the ``options`` object passed in the
```

Comment:
If you standardize this with the desciption of options used in ``findOne`` I think this would read better. As for now, I think it could use one little tweak: 
```suggestion
You can specify additional options in the ``options`` object passed as the
```